sematic/api/endpoints/external_resources.py

```diff
@@ -0,0 +1,53 @@
+# Standard Library
+from http import HTTPStatus
+from typing import Optional
+
+# Third-party
+import flask
+
+# Sematic
+from sematic.api.app import sematic_api
+from sematic.api.endpoints.auth import authenticate
+from sematic.api.endpoints.request_parameters import jsonify_error
+from sematic.db.models.external_resource import (
+    ExternalResource as ExternalResourceRecord,
+)
+from sematic.db.models.user import User
+from sematic.db.queries import (
+    get_external_resource_record,
+    save_external_resource_record,
+)
+
+
+@sematic_api.route("/api/v1/external_resources/<resource_id>", methods=["GET"])
+@authenticate
+def get_resource_endpoint(user: Optional[User], resource_id: str) -> flask.Response:
+    record = get_external_resource_record(resource_id=resource_id)
+    if record is None:
+        return jsonify_error(
+            "No such resource: {}".format(resource_id), HTTPStatus.NOT_FOUND
```

Comment:
Nit to standardize with the rest of the codebase.
```suggestion
            f"No such resource: {resource_id}", HTTPStatus.NOT_FOUND
```